core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -295,4 +328,23 @@ private void closeBlockInStream(BlockInStream stream) throws IOException {
       }
     }
   }
+
+  private void handleRetryableException(BlockInStream stream, IOException e, CountingRetry retry)
+      throws IOException {
+    if (!retry.attemptRetry()) {
```

Comment:
nit: could move this to the call sites to keep the retry counting logic in one place

Reply:
Done.